scripts/west_commands/runners/openocd.py

```diff
@@ -168,6 +168,7 @@ def do_flash(self, **kwargs):
                ['-c', self.verify_cmd + ' ' + hex_name] +
                post_verify_cmd +
                ['-c', 'reset run',
+                '-c', 'mww 0xE000EDF0 0xA05F0000', #32984
```

Comment:
Thanks for this. No comments on the register writes themselves, but thoughts on getting this out of draft:

To abstract this properly, I think we need to either:

1. move these into one of the existing command line hooks (e.g. the one that ends up in `post_verify_cmd`), or
2. create a new command line hook

After that, we can either:

1. make a `boards/common/openocd-cortex-m.board.cmake` file that sets this up appropriately, and include it from all the right board.cmake files (not forgetting to update `openocd-nrf5.board.cmake` too), or
2. modify openocd.board.cmake to check Kconfig variables for Cortex-M and do it automatically the way it already looks at `CONFIG_X86` and `CONFIG_ARC` to decide if it uses `load_image` or `flash write_image erase`

Based on the way you've written this PR, it looks like you'll need to add a new command line option since this doesn't appear next to any of the existing `_cmd` lists here or below. I think modifying `openocd.board.cmake` instead of creating a new file will be better and easier.

Reply:
I'm also kind of surprised nobody noticed this yet. Is there a standard openocd command for this that doesn't require touching the register directly?

Reply:
> I'm also kind of surprised nobody noticed this yet. Is there a standard openocd command for this that doesn't require touching the register directly?

Not that we are aware of.  There's nothing in the code that is obvious.

Reply:
I sent a patch request upstream: http://openocd.zylin.com/#/c/6093/1 to change the default mode.